setup.cfg

```diff
@@ -0,0 +1,24 @@
+[flake8]
+max-line-length = 120
+show-source = true
+ignore =
```

Comment:
Out of curiosity, what's the reason for ignoring this particular set of `flake8` warnings?

Reply:
They are super annoying to enforce and very common to ignore

Reply:
Ok. It's fine - just curious as it's different from my own default set. :)